src/components/organisms/AdminVenueView/components/PrettyLink/PrettyLink.tsx

```diff
@@ -1,17 +1,54 @@
-import React from "react";
+import React, { useCallback } from "react";
+import { Link, LinkProps, useHistory } from "react-router-dom";
+import classNames from "classnames";
+
+import { useKeyPress } from "hooks/useKeyPress";
 
 import "./PrettyLink.scss";
 
-interface PrettyLinkProps {
-  title: string;
-  onClick: () => void;
+const HANDLED_KEY_PRESSES = ["Space", "Enter"];
+
+interface PrettyLinkProps extends Omit<LinkProps, "to"> {
+  className?: string;
+  onClick?: () => void;
+  title?: string;
+  to?: string;
```

Comment:
I think the behaviour (both expected and actual) is unclear if someone provides both `onClick` and `to`. We could probably coax typescript into catching this for us. Maybe `PrettyLinkWithOnClickProps` and `PrettyLinkWithToProps` and the function takes one or the other?

Reply:
That could work. It would also mean to go a bit against current common standard of not making props be interfaces but type aliases... or worse, add `extends`...

In general, can just decide to give one of them priority if both are provided and/or throw an error, but then...

I can always consider a scenario where `to` provides the link to go and `onClick` deals with a counting function, one that just adds and saves to DB, not necessarily do navigation by itself.

Reply:
So long as the behaviour is defined *and* clear. Either error OR handle it then I'm happy :)

Reply:
cleared up @ 033499f85175b3f9f5e2473cc327a814db5bce63